src/core/scene/a-scene.js

```diff
@@ -124,6 +124,7 @@ module.exports = registerElement('a-scene', {
         if (this.systems[name]) { return; }
         system = this.systems[name] = new systems[name](this);
         system.init();
+        system.update({});
```

Comment:
We call `update` in the [component constructor](https://github.com/aframevr/aframe/blob/master/src/core/component.js#L38). Should we do the same for systems?